ox-typst.el

```diff
@@ -1165,22 +1165,24 @@ Return PDF file name or raise an error if it couldn't be produced."
 
 ;; Citation Exporter
 (defun org-typst-export-bibliography (_keys files style properties _backend com)
-  (let ((dir (file-name-parent-directory (plist-get com :input-file)))
-        (title (plist-get properties :title)))
-    (format "#bibliography(%s%s(%s))"
+  (let* ((dir (file-name-parent-directory (plist-get com :input-file)))
+         (title (plist-get properties :title))
+         (rel-fs (mapcar (lambda (f) (org-typst--as-typst-path (file-relative-name f dir)))
+                         files)))
+    (format "#bibliography(%s%s\"%s\")"
             (and style (format "style: \"%s\", " style))
             (if title (format "title: %s, " (org-typst--as-string title)) "")
-            (mapconcat (lambda (f) (org-typst--as-string (file-relative-name f dir)))
+            (mapconcat #'identity
                        files
                        ", "))))
```

Comment:
You can provide `org-typst--as-typst-path` with the file, it will do the rest. Afterwards, you need to put the identifier/placeholder generated by `org-typst--as-typst-path` into the document.

I accidentally created the tests before checking that. You have to re-generate the tests. 

1. Go into "tests/test.el"
2. Evaluate the file
3. Goto the test Org file 
4. Export to PDF or Typst

The resulting Typst should have references with like `org0000000`.

```suggestion
         (rel-fs (mapcar #'org-typst--as-typst-path files)))
    (format "#bibliography(%s%s%s)"
            (and style (format "style: \"%s\", " style))
            (if title (format "title: %s, " (org-typst--as-string title)) "")
            (mapconcat #'identity rel-fs ", "))))
```